dataservice/api/sample/models.py

```diff
@@ -0,0 +1,19 @@
+from dataservice.extensions import db
+from dataservice.api.common.model import Base
+
+
+class Sample(db.Model, Base):
+    """
+    Sample entity.
+    :param _id: Unique id assigned by RDBMS
+    :param kf_id: Unique id given by the Kid's First DCC
+    :param external_id: Name given to sample by contributor
+    :param composition : composition of the sample
+    :param tissue_type: Either Normal or Tumor
+    """
+    __tablename__ = "sample"
+    external_id = db.Column(db.String(32))
+    tissue_type = db.Column(db.String(32))
```

Comment:
Maybe these should be longer? Not sure on the requirements and what kinds of headaches it could cause if we get something longer than 32 chars some time.

Reply:
The kf-dictionary tissue_type has values like "Normal", "Abnormal" etc. So maybe its ok? But @allisonheath would know best.

https://github.com/kids-first/kf-dictionary/blob/514fee43d816e98e6bfae2427c4207c53785710f/gdcdictionary/schemas/sample.yaml#L483

Reply:
In general with postgres there's no longer any reason to limit the character fields unless you want to for QC purposes: https://stackoverflow.com/questions/4848964/postgresql-difference-between-text-and-varchar-character-varying

So I'd suggest using the `Text` type here in the spirit of not backing ourselves into a corner with variable data.